exercises/two-bucket/.meta/solutions/two_bucket.rb

```diff
@@ -68,6 +72,22 @@ def start_from_second
     end
   end
 
+  def fill_other_bucket
+    if first_bucket_empty?
+      fill_first_bucket
+    else
+      fill_second_bucket
+    end
+  end
+
+  def other_bucket_matches_goal
+    if @start_bucket == 'one'
+      return @second_size == @goal
+    else
+      return @first_size == @goal
```

Comment:
Redundant `return` used here.  The last statement of the method is the if statement, and therefore will be returned.